homeassistant/components/statsd.py

```diff
@@ -57,8 +57,16 @@ def statsd_event_listener(event):
         except ValueError:
             return
 
+        states = dict(state.attributes)
+
         _LOGGER.debug('Sending %s.%s', state.entity_id, _state)
-        statsd_client.gauge(state.entity_id, _state, sample_rate)
+        statsd_client.gauge("%s.state" % state.entity_id, _state, sample_rate)
```

Comment:
Hmmm... good point. May see if it's possible to put a value mid topic chain. If not will need to come up with a practical solution.